src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -276,7 +276,7 @@ def post_process(self, outputs, target_sizes, offset_mapping, bbox):
         results = []
 
         for s, l, b, o, (height, width) in zip(scores, labels, boxes, offsets, target_sizes):
-            # only keep start of a particular word (i.e. offset start == 0) (e.g. [San, Fran, ##cis, ##co] -> [San])
+            # only keep start of a particular word (i.e. offset start == 0) (e.g. [San, Fran, ##cis, ##co] -> [San, Fran])
```

Comment:
Are you sure ?
It depends on the boxes boundary of the OCR no ? If OCR outputs [SAN] [Francisco] then yes,
but if it outputs [San Francisco] it feels like the previous commentary was correct.

No ?

Reply:
Offsets != words....

Reply:
Yes you're right @Narsil, it depends on the OCR. Good catch.

Reply:
Reverted the comment to the previous (correct) version 
https://github.com/huggingface/transformers/blob/3825a4175a184f643085d978c5bafd4518f40fe0/src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py#L277-L280